icons/square/48/duckduckgo.svg

```diff
@@ -0,0 +1,28 @@
+<svg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can you resave the file with using 2 spaces as indentation? Like all other files :+1:

Reply:
Other files use 1 space indentation - I started using 1 tab because I can set my editor tab length to any value.

Reply:
Also, here is a solution to having different indentation on file system and git repository:
https://stackoverflow.com/questions/2316677/can-git-automatically-switch-between-spaces-and-tabs#2318063

Reply:
This can be set in the Inkscape (Scour) optimisation presets as well.